### PR TITLE
Kalradev/extension/graphql

### PR DESCRIPTION
**Pull request by 0xfirefist**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: handlers/plugin_handlers.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	switch path {
+	case "/plugins/load":
+		h.loadPlugins(w, req, provider)
+	default:
+		plugins[path].ServeHTTP(w, req)
+	}
+}
+
+func (h *Handler) loadPlugins(w http.ResponseWriter, r *http.Request, provider models.Provider) {
+	plug, err := plugin.Open("/home/devkalra/Desktop/layer5labs/meshery-extensions/graphql/graphql-meshmap.so")
```
Comment (leecalcote) [inline handlers/plugin_handlers.go:32]: Reference needs corrected.

File: router/server.go [hunk 2]
```diff
@@ -20,6 +20,9 @@ type Router struct {
 func NewRouter(ctx context.Context, h models.HandlerInterface, port int) *Router {
 	gMux := mux.NewRouter()
 
+	gMux.PathPrefix("/plugins/").
```
Comment (leecalcote) [inline router/server.go:23]: @kalradev will you update the Meshery Extensibility doc with this proposed path? It'll be easier to review in context of other extension points in the design spec.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: handlers/plugin_handlers.go (hunk 1 above)
Comment (kalradev) [reply handlers/plugin_handlers.go:32]: Yup, will do that we are trying to debug this for now. Before merging to master we will change this.

File: router/server.go (hunk 2 above)
Comment (kalradev) [reply router/server.go:23]: Sure, After completing this will do

Comment (leecalcote) [reply router/server.go:23]: Sounds real good.

**Pull request closed without merging**